frontend-react/src/App.js

```diff
@@ -11,19 +12,23 @@ import { createBrowserHistory } from 'history';
 import rtl from 'jss-rtl';
 import { create } from 'jss';
 import CssBaseline from '@material-ui/core/CssBaseline';
+import { Grid, IconButton, Link } from "@material-ui/core";
```

Comment:
I believe we've agree to use single quotes in imports. Please change also in other places. I promise to work on and add eslint rules for this project asap :-)

Reply:
Oh, sorry! I often use editor autocomplete when I'm writing imports but the editor always suggests the code with double quotes and I forget to fix that.